### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Anton20HD**
Added some running instructions, but I am not sure if i got all of them.

### CODE REVIEW

**Review by RobertMili — DISMISSED** [review]
Great work

File: README.md [hunk 1]
```diff
@@ -37,3 +53,50 @@
     <img src="https://img.shields.io/github/languages/top/fungover/haze" alt="Language workflow badge"/>
 </div>
 </section>
+
+
```
Comment (jLereback) [inline README.md:57]:
```suggestion

# Haze
Key-Value database that talks RESP

```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+
+
+
+
+
+
+
+
+
```
Comment (jLereback) [inline README.md:16]:
```suggestion
```

**Review by jLereback — CHANGES_REQUESTED** [review]
Great work! I just have a tiny change to separate the instruction from the workflow badges :hugs:

**Pull request merged**